sematic/ui/src/types/Types.tsx

```diff
@@ -261,17 +263,7 @@ function ListValueView(props: ValueViewProps) {
           </TableCell>
           <TableCell>{props.valueSummary["length"]}</TableCell>
         </TableRow>
-        <TableRow>
-          <TableCell>
-            <b>Excerpt</b>
-          </TableCell>
-          <TableCell>
-            {summary} and{" "}
-            {props.valueSummary["length"] -
-              props.valueSummary["summary"].length}{" "}
-            more items.
-          </TableCell>
-        </TableRow>
```

Comment:
Any reason for this change?
Do you need to rebase to `main` perhaps?

Reply:
Good that you catch that, thanks 🙏. Yeah, it was a mistake (probably during some conflict while rebasing). I put it back 🙂.